corehq/apps/style/static/style/js/hq-bug-report.js

```diff
@@ -79,6 +80,9 @@ $(function () {
     function hqwebappBugReportSucccess(data) {
         isBugReportSubmitting = false;
         $hqwebappBugReportForm.find("button[type='submit']").bootstrapButton('success').removeClass('btn-primary btn-danger').addClass('btn-success');
+        $hqwebappBugReportModal.one('hidden.bs.modal', function () {
```

Comment:
nit: could just be '$hqwebappBugReportModal.one('hidden.bs.modal', resetForm)' though i guess that means you get args passed in